diff -dur lxdm-0.5.1.bak/src/lxdm.c lxdm-0.5.1/src/lxdm.c
--- lxdm-0.5.1.bak/src/lxdm.c	2015-02-14 04:07:16.000000000 +0100
+++ lxdm-0.5.1/src/lxdm.c	2015-10-12 21:08:11.385134263 +0200
@@ -1588,6 +1588,10 @@
 		if(strstr(pw->pw_shell, "nologin"))
 			continue;
 
+		// PLD Linux: skip users with homedir = /home/services
+		if(!strncmp(pw->pw_dir,"/home/services/",15))
+			continue;
+
 		ret = FALSE;
 		setusershell();
 		while ((valid_shell = getusershell()) != NULL) {
