--- lxdm-0.5.3/src/ui.c.orig	2023-05-06 08:01:13.387616230 +0200
+++ lxdm-0.5.3/src/ui.c	2023-05-06 08:02:24.333898548 +0200
@@ -244,6 +244,7 @@ void ui_prepare(void)
 			first=FALSE;
 		}
 		pw=getpwnam("lxdm");endpwent();
+		setuid(0);
 		ret = g_spawn_async_with_pipes(NULL, argv, NULL,
 				   G_SPAWN_SEARCH_PATH | G_SPAWN_DO_NOT_REAP_CHILD,greeter_setup, pw,
 				   &greeter, greeter_pipe + 0, greeter_pipe + 1, NULL, NULL);
