--- lxdm-0.2.0/src/ui.c.old	2010-08-02 09:13:00.000000000 +0200
+++ lxdm-0.2.0/src/ui.c	2010-08-02 09:13:40.000000000 +0200
@@ -487,6 +487,8 @@ void ui_prepare(void)
         if( greeter > 0 && kill(greeter, 0) == 0 )
             return;
 
+
+        setuid(0);
         ret = g_spawn_async_with_pipes(NULL, argv, NULL,
                                        G_SPAWN_SEARCH_PATH | G_SPAWN_DO_NOT_REAP_CHILD, greeter_setup, 0,
                                        &greeter, greeter_pipe + 0, greeter_pipe + 1, NULL, NULL);
